src/qobj/arithmetic_and_attributes.jl

```diff
@@ -8,6 +8,7 @@ export sqrtm, sinm, cosm
 export ptrace
 export tidyup, tidyup!
 export get_data, get_coherence
+export permute
```

Comment:
Why do we export `permute` also in this file?

Reply:
this should not be there, i was exporting it from there before i saw it made more sense (at least to me) to make the `src/qobj/tensor_functions.jl` file